src/services/ui/src/pages/actions/WithdrawPackage.tsx

```diff
@@ -0,0 +1,229 @@
+import { Navigate, useNavigate, useParams } from "react-router-dom";
+import { Button } from "@/components/Inputs";
+import { ConfirmationModal } from "@/components/Modal/ConfirmationModal";
+import { useEffect, useState } from "react";
+import { Action, ItemResult, withdrawPackageSchema } from "shared-types";
+import { FAQ_TARGET, ROUTES } from "@/routes";
+import { PackageActionForm } from "./PackageActionForm";
+import { ActionFormIntro, PackageInfo } from "./common";
+import { z } from "zod";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import * as I from "@/components/Inputs";
+import { Link } from "react-router-dom";
+import { Alert, LoadingSpinner } from "@/components";
+import { buildActionUrl } from "@/lib";
+import { useGetUser } from "@/api/useGetUser";
+import { useSubmissionService } from "@/api/submissionService";
+
+const withdrawPackageFormSchema = withdrawPackageSchema(
+  z.array(z.instanceof(File))
+);
+type WithdrawPackageFormSchema = z.infer<typeof withdrawPackageFormSchema>;
+type UploadKey = keyof WithdrawPackageFormSchema["attachments"];
+type AttachmentRecipe = {
```

Comment:
Opportunity to extract and re-use these types since forms with attachments all get set up pretty much the same. :) Yay new submission service!